cdap-app-fabric/src/main/java/io/cdap/cdap/metadata/LineageHTTPHandler.java

```diff
@@ -301,7 +301,7 @@ public long getEnd() {
   private Constants.FieldLineage.Direction parseDirection(String directionStr) throws BadRequestException {
     try {
       return Constants.FieldLineage.Direction.valueOf(directionStr.toUpperCase());
-    } catch (IllegalArgumentException e) {
+    } catch (NullPointerException | IllegalArgumentException e) {
```

Comment:
Is that mean `directionStr` can be null? Please annotate with `@Nullable` if that's the case.

Reply:
Fixed